doc/api/cli.md

```diff
@@ -455,6 +455,14 @@ added: v0.8.0
 
 Silence deprecation warnings.
 
+### `--no-experimental-warnings`
+<!-- YAML
+added: REPLACEME
+-->
+
+Silence all experimental process warnings. These are emitted when using
+features which are considered experimental.
```

Comment:
```suggestion
features which are experimental.
```

Reply:
Might want to add an explicit warning against using in production and/or a link to the definition of _experimental_ elsewhere in the docs. But if that doesn't happen hear, it can happen in a subsequent PR. Not a blocking comment.